### ISSUE TITLE
[Feature Request] Enhanced Visibility of Replies

### ISSUE THREAD

**Issue by crimsonleaf363** [nostur-com/nostur-ios-public#11]
It's difficult to distinguish between replies and the original post.

### **Feature**

- Add a recognizable change to replies so they are easier to recognize from the original post. You could consider changing the hue, shade, or adding a gradient.

### **Benefits**

- Less confusion: users can more easily see replies to specific posts.
- Increased Engagement: conversations are easier to follow

**Comment by fabianfabian** [comment]
This is probably only when using full-width images right? or are they also difficult to distinguish without that?

**Comment by crimsonleaf363** [comment]
I think they are difficult to distinguish, regardless of it's text or an image. I'm using full width images, but I haven't noticed an effect.

**Comment by fabianfabian** [comment]
Yes if you disable full width images its very clear because of the thread line:

![image](https://github.com/nostur-com/nostur-ios-public/assets/343640/ad7f763a-046c-4228-a930-4b4a43c523cf)

I'm not sure what the best solution is for with full width images enabled, personally I don't use that but I'll figure something out.

**Comment by crimsonleaf363** [comment]
You’re right. It was because of the full width images. I never noticed because I changed my settings before really using the app.

**Comment by fabianfabian** [comment]
I made connecting thread lines always visible, regardless of full-width image setting. This solves the main problem. Full-width images still apply but only to main/root posts.
